docs/migration/dapp_migration.md

```diff
@@ -31,6 +31,7 @@ For starters, here's a sampling of exciting perks dapps with get with the Nitro
 - **Retryable Tickets**: 
     - The submission cost is now enforced in the L1 inbox and checked against the L1 transaction's `msg.value`; contracts shouldn't rely on funds pooled in the L2 destination to cover this cost.
     - For the redemption of retryable tickets, the calculation of the L2 transaction ID changed and differs between different redeem attempts (i.e. after failed attempts). See [arbitrum-sdk](https://github.com/offchainlabs/arbitrum-sdk/tree/c-nitro) for a reference implementation on the new client-side flow. 
+    - Retryable ticket now aren't redeemed in the same block as when the `redeem` function is called. The user's transaction causes the retryable to be scheduled to be redeemed in a subsequent block.
```

Comment:
I think it's scheduled to be the next tx in the same block. Are you sure about this?

Reply:
trust you more, will change it